packages/workspace/src/core/project-graph/daemon/exec/stop.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * NOTE: This file is intended to be executed as a script by utilities found in ./index.ts
+ */
+import { logger } from '@nrwl/devkit';
+import { deleteDaemonJsonCache, readDaemonJsonCache } from '../cache';
+import { stopServer } from '../server';
+
+(async () => {
+  try {
+    await stopServer();
+
+    // Clean up any orphaned background process and clear cached metadata
+    const cachedDaemonJson = await readDaemonJsonCache();
+    if (cachedDaemonJson) {
+      if (cachedDaemonJson.backgroundProcessId) {
+        process.kill(cachedDaemonJson.backgroundProcessId);
```

Comment:
Yeah it hopefully shouldn't matter because the children are not doing anything to handle/block exit on SIGTERM, but I think I may as well upgrade it to SIGKILL